lint-baseline.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<issues format="6" by="lint 8.8.2" type="baseline" client="gradle" dependencies="false" name="AGP (8.8.2)" variant="all" version="8.8.2">
+
+    <issue
+        id="NewApi"
+        message="Call requires API level 33 (current min is 21): `java.lang.ref.Cleaner#create`"
+        errorLine1="    val cleaner = java.lang.ref.Cleaner.create()"
+        errorLine2="                                        ~~~~~~">
+        <location
+            file="build/generated/uniffi/mozilla/telemetry/glean/internal/glean.kt"
+            line="2613"
+            column="41"/>
+    </issue>
+
+    <issue
+        id="NewApi"
+        message="Call requires API level 33 (current min is 21): `java.lang.ref.Cleaner#register`"
+        errorLine1="        JavaLangRefCleanable(cleaner.register(value, cleanUpTask))"
+        errorLine2="                                     ~~~~~~~~">
+        <location
+            file="build/generated/uniffi/mozilla/telemetry/glean/internal/glean.kt"
+            line="2616"
+            column="38"/>
+    </issue>
+
+    <issue
+        id="NewApi"
+        message="Call requires API level 33 (current min is 21): `java.lang.ref.Cleaner.Cleanable#clean`"
+        errorLine1="    override fun clean() = cleanable.clean()"
+        errorLine2="                                     ~~~~~">
+        <location
+            file="build/generated/uniffi/mozilla/telemetry/glean/internal/glean.kt"
+            line="2622"
+            column="38"/>
+    </issue>
+
+</issues>
```

Comment:
What's this?

Reply:
The "baseline" file `./gradlew updateLintBaseline` creates. Means a new run won't trigger on those lints and accepts them.